build.gradle.kts

```diff
@@ -522,6 +523,50 @@ task("updateCargoOptions") {
     }
 }
 
+task("updateLints") {
+    doLast {
+        val lints = JsonSlurper().parseText("python3 fetch_lints.py".execute("scripts", print = false)) as List<Map<String, *>>
+
+        fun writeLints(path: String, lints: List<Map<String, *>>, variableName: String, prefix: String = "") {
+            val file = File(path)
+            val items = lints.sortedWith(compareBy({ !(it["group"] as Boolean) }, { it["name"] as String})).joinToString(
```

Comment:
Maybe we can extract `it["group"] as Boolean` into local function? Something like
```
fun Map<String, *>.isGroup(): Boolean = get("group") as Boolean
```

And the same for `name`
Just to make code a little bit clearer and avoid code duplication - we use such casts in two places